CHANGELOG.md

```diff
@@ -6,54 +6,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
-## [v10.1.1] 2022-03-04
```

Comment:
Git/github merge kept the commit back to main because the OG commit was squashed and it couldn't match up the changeset.   Easier to remove it here than in the mergeback.   This is just removing duped content.